Packs/DragosWorldview/Integrations/DragosWorldview/DragosWorldview.yml

```diff
@@ -0,0 +1,97 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: 'Dragos Worldview'
+  version: -1
+configuration:
+- defaultvalue: https://portal.dragos.com/api/v1/
+  display: Server URL (e.g. https://portal.dragos.com/api/v1/)
+  name: url
+  required: true
+  type: 0
+- display: API Token
+  name: apitoken
+  required: true
+  type: 4
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- defaultvalue: 3 days
+  display: First fetch time
+  name: first_fetch
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- defaultvalue: "1"
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- additionalinfo: The Traffic Light Protocol (TLP) designation to apply to indicators
+  display: Traffic Light Protocol Color
+  name: tlp_color
+  options:
+  - RED
+  - AMBER
+  - GREEN
+  - WHITE
+  required: false
+  type: 15
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- defaultvalue: "50"
+  display: Fetch Limit
+  name: max_fetch
+  required: false
+  type: 0
+description: 'Custom integration designed to pull in reports from the Dragos Worldview
+  API as incidents '
+display: 'Dragos Worldview'
+name: 'Dragos Worldview'
+script:
+  commands:
+  - arguments:
+    - description: Report serial number to get indicators from, if no serial number
+        provided command will retrieve all indicators from the last 48 hours
+      name: serial
+    description: Get Indicators from the Dragos WorldView API
+    name: dragos-get-indicators
+  - arguments:
+    - description: Serial number for the report to retrieve
+      name: serial
+      required: true
+    description: Get the report file from the given serial number
+    name: dragos-get-full-report
+  - arguments:
+    - description: Serial number of the report from which to get the file
+      name: serial
+      required: true
+    description: Get csv file with indicators from a given report
+    name: dragos-get-ioc-csv
+  - arguments:
+    - description: Serial number of the report from which to retrieve the file
+      name: serial
+      required: true
+    description: Get the stix2 json bundle of indicators from a given report
+    name: dragos-get-stix2
+  dockerimage: demisto/python3:3.10.4.29342
+  isFetchSamples: true
+  isfetch: true
+  runonce: false
+  script: ''
+  subtype: python3
+  type: python
+fromversion: 5.0.0
```

Comment:
please set the `fromversion` to be the version of your XSOAR server you tested this pack with